README.rst

```diff
@@ -265,11 +266,12 @@ Red Hat family
 ~~~~~~~~~~~~~~
 
 - Amazon Linux 2012.3 and later
-- CentOS 6/7
+- Amazon Linux 2
+- CentOS 6/7/8
 - Cloud Linux 6/7
-- Fedora 27/28 (install latest stable from standard repositories)
+- Fedora 30/31 (install latest stable from standard repositories)
```

Comment:
I think we need to update distro EOL checks within the bootstrap script to reflect what is documented here:
https://github.com/saltstack/salt-bootstrap/blob/f6d5027ea7997342e9d70c44628fa5facb75a865/bootstrap-salt.sh#L1617

Reply:
https://github.com/saltstack/salt-bootstrap/issues/1400